Add quick links to home page (#286)
ocf/ocfstatic: docs/services/webhostingsolutions.md

---

> title: "Web hosting solutions at OCF"

Reviewer: Can the title be improved? Maybe just "Web hosting solutions"

Author: Sure, that works. I wanted to emphasize that this is the overview page while the others are the specific pages for each type.

Reviewer: Could we title it "What type of web hosting do I need?" Of course, I realize this isn't necessarily consistent with the rest of the documentation since none of the other pages are titled with questions.

Author: I think the break in consistency could be helpful in this context since it would grab the user's attention

---

## What type of web hosting do I need?

The OCF offers two main types of hosting, personal hosting and group hosting. If you have a personal account (which most people have unless you have requested a group account), then your main option is personal hosting. If you have a group account for a Registered Student Organization, then you have a couple options for your hosting solutions, which are listed in the group hosting section.

## Hosting your personal site

> Head on over to [the documentation](/docs/services/web/) to get started!

Reviewer: ```suggestion
See [personal hosting documentation](/docs/services/web/) to get started!
```

## Hosting your group's site

> You should start with our [virtual hosting solution](/docs/services/vhost/) if you are just getting started with group hosting at the OCF. If you have applied for application hosting and was approved, head over to the [app hosting documentation](/docs/services/webapps/).

Reviewer: ```suggestion
You should start with our [virtual hosting documentation](/docs/services/vhost/) if you are just getting started with group hosting at the OCF. If you have applied and been approved for application hosting, head over to the [app hosting documentation](/docs/services/webapps/).
```
